src/main/java/woowacourse/shoppingcart/ui/CartItemController.java

```diff
@@ -1,47 +1,64 @@
 package woowacourse.shoppingcart.ui;
 
+import java.util.List;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.validation.annotation.Validated;
-import org.springframework.web.bind.annotation.*;
-import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
-import woowacourse.shoppingcart.domain.Cart;
-import woowacourse.shoppingcart.domain.Product;
-import woowacourse.shoppingcart.dto.Request;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import woowacourse.auth.config.AuthenticationPrincipal;
 import woowacourse.shoppingcart.application.CartService;
-
-import java.net.URI;
-import java.util.List;
+import woowacourse.shoppingcart.dto.CartProductRequest;
+import woowacourse.shoppingcart.dto.CartQuantityRequest;
+import woowacourse.shoppingcart.dto.CartResponse;
+import woowacourse.shoppingcart.dto.LoginCustomer;
+import woowacourse.shoppingcart.dto.Request;
 
 @RestController
-@RequestMapping("/api/customers/{customerName}/carts")
+@RequestMapping("/customers/carts")
```

Comment:
```suggestion
@RequestMapping("/customers/me/carts")
```
어떤 customers인지가 식별자 명시가 안되어있네요. 토큰기반으로 되어있어서 완벽하게 REST하게 지켜지진 않지만, 이렇게 대체할 수 있는 것 같아요.

Reply:
반영했습니다 감사합니다 😄